File: python/duck/Duck.py
class Duck:
    """this is a Duck class"""
    def __init__(self, name):
        self.name = name

    def quack(self):
        print(f'the {self.name} is quacking')

    def swim(self):
        print(f'the {self.name} is swimming')

    def display(self):
        print(f'I am a {self.name}')

File: java/duck/Duck.java
public class Duck {

    String name;

    public Duck(String name) { this.name = name; }

    public void quack() {
        System.out.println(name + " is quacking");
    }

    public void swim() {
        System.out.println(name + " is swimming");
    }

    public void display() {
        System.out.println("I am a " + name);
    }

}
